schema/queue/message/impl/RabbitMqMessageSubscriber.go

```diff
@@ -94,23 +84,12 @@ func (cs *CommonMessageSubscriber) ConfirmationHandler(msgDelivery amqp.Delivery
 	if handleErr != nil {
 		cs.Logger.Fatal().Err(handleErr).Msg("Can't Handle")
 	}
-	for _, group := range result.Groups {
-		for _, msg := range group.Messages {
-			switch msg.GetKind().(type) {
-			case *p_buff.AnyMessage_RawMessage:
-				msg := msg.GetRawMessage()
-				th2_message_subscribe_total.WithLabelValues(cs.th2Pin, msg.Metadata.Id.ConnectionId.SessionAlias, string(msg.Metadata.Id.Direction), RAW_MESSAGE_TYPE).Inc()
-			case *p_buff.AnyMessage_Message:
-				msg := msg.GetMessage()
-				th2_message_subscribe_total.WithLabelValues(cs.th2Pin, msg.Metadata.Id.ConnectionId.SessionAlias, string(msg.Metadata.Id.Direction), MESSAGE_TYPE).Inc()
-			}
-		}
-	}
+	metrics.UpdateMessageMetrics(result, th2_message_subscribe_total, cs.th2Pin)
```

Comment:
Could you please put this instruction before handling because we already receive a batch